suite-native/transactions/src/components/TransactionDetail/TransactionDetailValuesSheet.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { selectCoins } from '@suite-common/wallet-core';
+import { useFormatters } from '@suite-common/formatters/src';
+import { WalletAccountTransaction } from '@suite-common/wallet-types/src';
+import { Card, Table, Text, VStack } from '@suite-native/atoms/src';
+import { formatNetworkAmount, toFiatCurrency } from '@suite-common/wallet-utils';
+import { FiatCurrency } from '@suite-common/suite-config/src';
+
+import { TransactionDetailSheet } from './TransactionDetailSheet';
+
+type TransactionDetailValuesSheetProps = {
+    isVisible: boolean;
+    transaction: WalletAccountTransaction;
+    fiatCurrency: FiatCurrency;
+    handleSheetVisibility: () => void;
+};
+export const TransactionDetailValuesSheet = ({
+    isVisible,
+    handleSheetVisibility,
+    transaction,
+    fiatCurrency,
+}: TransactionDetailValuesSheetProps) => {
+    const { FiatAmountFormatter } = useFormatters();
+    const coins = useSelector(selectCoins);
+
+    const transactionInput = formatNetworkAmount(transaction.amount, transaction.symbol);
+    const transactionFee = formatNetworkAmount(transaction.fee, transaction.symbol);
+    const transactionTotal = formatNetworkAmount(
+        transaction.details.totalOutput,
+        transaction.symbol,
+    );
+
+    // historical rates
+    const fiatInputHistoryFormatted = FiatAmountFormatter.format(
```

Comment:
`fiatTransactionInputInitialAmount` maybe?

Reply:
history sounds bit weird

Reply:
Please get rid of all these variables and use them in render directly. And you should not need `toFiatCurrency` at all.